.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "python.testing.pytestArgs": [
+        "tests"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "editor.defaultFormatter": "ms-python.black-formatter"
+}
```

Comment:
Should we add a newline at the end of this file?

Reply:
Added it